### PR TITLE
Implement Container.getPlugin() for Forge mods.

### PR DESCRIPTION
**Pull request by JBYoshi**
[SpongeCommon](https://github.com/SpongePowered/SpongeCommon/pull/1357) | **SpongeForge**

### CODE REVIEW

File: src/main/java/org/spongepowered/mod/mixin/core/fml/common/registry/MixinNetworkRegistry.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import net.minecraftforge.fml.common.ModContainer;
+import net.minecraftforge.fml.common.network.IGuiHandler;
+import net.minecraftforge.fml.common.network.NetworkRegistry;
+import org.spongepowered.api.plugin.PluginContainer;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Redirect;
+import org.spongepowered.common.interfaces.IMixinContainer;
+
+@Mixin(NetworkRegistry.class, remap=false)
```
Comment (kashike) [inline src/main/java/org/spongepowered/mod/mixin/core/fml/common/registry/MixinNetworkRegistry.java]: `remap = false` (spacing)

**Review by kashike — CHANGES_REQUESTED** [review]

File: src/main/java/org/spongepowered/mod/mixin/core/fml/common/registry/MixinNetworkRegistry.java (hunk 1 above)
Comment (JBYoshi) [reply src/main/java/org/spongepowered/mod/mixin/core/fml/common/registry/MixinNetworkRegistry.java]: ...And the line I wrote doesn't even compile - I forgot `value =`. That's what I get for coding when I'm too pressed on time...

Comment (JBYoshi) [reply src/main/java/org/spongepowered/mod/mixin/core/fml/common/registry/MixinNetworkRegistry.java]: Fixed.

**Comment by JBYoshi** [comment]
Closed in https://github.com/SpongePowered/SpongeForge/commit/e15410874c1dadc78a1a487e6f2d457fa31ab2d7.

**Pull request closed without merging**